index.rst

```diff
@@ -1,83 +1,144 @@
+
+* Finally, take the logarithm of 1 plus the difference image source significance divided by the square root of the reference flux.
+  Dividing by the square root of the reference flux makes the quantity unitless, but the remainder of this step is fairly arbitrary and simply makes the output more easily human readable.
+
+
+In code, the above looks like::
+
+  ref_src = butler_ref.get("src", dataId=_id)      # load the source list with a butler
+  flux_ref = ref_src[fluxrefKey]                   # Extract a flux measurement, e.g. "base_PsfFlux_flux"
+  flux_ref = flux_ref[~ref_src[flagrefKey]]        # Remove flagged sources 
+  flux_ref = numpy.clip(flux_ref, 1.0, saturation) # Clip flux at saturation threshold (65000 counts)
+  flux_ref = np.sum(flux_ref)                   
+
+  dia_src = butler_dia.get("deepDiff_diaSrc", dataId=_id)                                  # load the difference image source list with a butler
+  combined_flux = (numpy.abs(dia_src[posFluxKey]) + numpy.abs(dia_src[negFluxKey]))        # Add the absolute value of positive and negative
+  significance = numpy.clip(combined_flux, 1.0, saturation) / numpy.abs(dia_src[sigmaKey]) # Clip at saturation threshold, and divide by the error in the flux measurement
+  significance = significance[~dia_src[flagKey]]                                           # Remove flagged sources
+  metric = numpy.log(1.0 + numpy.sum(metric) / numpy.sqrt(flux_ref))
+
+Finally, calculating the above dipole severity metric for the "Faint" and "Bright" simulations gives the following:
+
+Dipole Severity Metric Plots
+----------------------------
+
+.. figure:: /_static/Dipole_severity_metric_plot_test6.png
+   :name: fig-dipole_severity6
+   :target: https://github.com/lsst-dm/dmtn-019/tree/master/_static/Dipole_severity_metric_plot_test6.png
+
+   Plot of the dipole severity metric for the "Faint" simulations. 
+
+.. figure:: /_static/Dipole_severity_metric_plot_test8.png
+   :name: fig-dipole_severity8
+   :target: https://github.com/lsst-dm/dmtn-019/tree/master/_static/Dipole_severity_metric_plot_test8.png
+
+   Plot of the dipole severity metric for the "Bright" simulations.
+
+Conclusions
+===========
+
+The dipole severity metric plotted above shows that g-band is considerably worse affected than either u- or r-band, which matches what we would expect from visual inspection of Figures 1 and 2. 
+I don't have a good explanation for why u- and r-band have such extreme scatter in the "Bright" simulation, except that it appears to be a real effect of the image differencing pipeline.
+In Figure 4 above, the "Bright" u-band field shows a pronounced dip at an elevation angle of 45 degrees only on the rising side of zenith.
+As seen in Figure 5 below, there really do appear to be far fewer dipoles when subtracting that particular observation, suggesting that the scatter in the plot of the metric above is capturing actual significant differences in the performance of image differencing.
```

Comment:
I don't think it's due to registration, since these are simulation images that on input are registered perfectly. Calibration does report shifts of up to 1 milliarcsecond, but that's so small it shouldn't matter. 
I agree with this and the previous comment that the plots look noisy for u- and r-bands, but I think that is a result of inconsistent success of our existing image differencing algorithms.